setup.py

```diff
@@ -12,11 +12,11 @@
 PACKAGES = [
     'crowdstrike.foundry.function',
 ]
-PYTHON_VERSION = '>=3.9.0,<3.10.0'
+PYTHON_VERSION = '>=3.9.0'
 SETUP_REQUIRES = [
     'setuptools',
 ]
-VERSION='0.3.0'
+VERSION = '0.4.0'
```

Comment:
I believe this will be `0.5.0`, we already have a `0.4.0`. The `setup.py` wasn't updated last release.